src/mon/PGMonitor.cc

```diff
@@ -1256,11 +1256,14 @@ epoch_t PGMonitor::send_pg_creates(int osd, Connection *con, epoch_t next)
              << dendl;
     last = q->first;
     for (set<pg_t>::iterator r = q->second.begin(); r != q->second.end(); ++r) {
+      pg_stat_t &st = pg_map.pg_stat[*r];
+      if (st.parent_split_bits)
+	continue;
```

Comment:
can we skip the split PGs in `PGMap::stat_pg_add()` at the very beginning? the `pg_map.pg_stat` is always updated at that moment.